components/eam/src/physics/cam/polar_avg.F90

```diff
@@ -179,6 +180,9 @@ subroutine polar_average3d(nlev, field)
            end if
         enddo
      end do
+
+n_pole=0
+s_pole=0
```

Comment:
@aldivi and @evasinha: Do we need these changes?

Reply:
@singhbalwinder I don't know. maybe they were removed from v2 because they were not necessary? I am not familiar with this code